gcb/presubmit.sh

```diff
@@ -68,3 +68,21 @@ bazel_run build \
 
 # Run all the tests.
 bazel_run test //src/...
+
```

Comment:
You might want to fix the comment above this line, as with the addition of the invocation for auth logic it does not run all of the tests.

Reply:
Done